packages/@ama-sdk/schematics/schematics/typescript/shell/templates/base/readme.md

```diff
@@ -140,6 +139,8 @@ Simple example:
     properties:
       departureDateTime:
         type: string
+        x-local-timezone:
+        description: If this vendor extension is present send dates without their timezone
```

Comment:
If the `description` is for `x-local-timezone` I think we need an extra tab

Reply:
Nope, it is for the departure date time object